Wireframe/index.html

```diff
@@ -4,10 +4,50 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Wireframe</title>
-    <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" href="css/style.css">
   </head>
   <body>
-    <!-- Add your HTML markup here -->
-    <!-- Remember: Use semantic HTML tags like <header>, <main>, <nav>, <footer>, <section> etc -->
+
+    <header>
+      <h1>Understanding Git</h1>
+      <p>A Brief Overview of Git and Its Key Concepts</p>
+  </header>
+
+  <main>
+      <!-- Full-width blog post box -->
+      <section class="blog-post full-width">
```

Comment:
What led you to choose section here instead of article? Can you talk through your decision?

Reply:
@SallyMcGrath ,I chose to use section instead of article because all the blog posts are related to the same topic about Git.

Using section helps group this content together and shows that they are connected. Each blog post is part of a bigger idea, so I thought section was the right choice.

If each post were meant to stand alone as separate pieces, then article would have been better. I see that now and will keep this in mind for future projects. Thank you for the feedback!